pynamodb/exceptions.py

```diff
@@ -14,6 +14,16 @@ def __init__(self, msg=None, cause=None):
         self.cause = cause
         super(PynamoDBException, self).__init__(self.msg)
 
+    def get_cause_response_code(self):
+        if hasattr(self.cause, 'response'):
```

Comment:
nit -
```python
return getattr(self.cause, 'response', {}).get('Error', {}).get('Code')
```

Reply:
Also, seems rather lightweight and no side effects, maybe make it a property?

Reply:
sounds good!

Reply:
updated